apps/site/lib/site_web/templates/trip_plan/_fare_calculator.html.eex

```diff
@@ -66,25 +66,27 @@
 </div>
 
 
-<div class="m-trip-plan-farecalc__table">
-  <div class="m-trip-plan-farecalc__table-cell m-trip-plan-farecalc__subheader">
-    Monthly Passes
-  </div>
-  <div class="m-trip-plan-farecalc__table-cell m-trip-plan-results__itinerary-fare-title m-trip-plan-farecalc__header">
-    Base
-  </div>
-  <div class="m-trip-plan-farecalc__table-cell m-trip-plan-results__itinerary-fare-title m-trip-plan-farecalc__header">
-    Reduced
-  </div>
-  <div class="m-trip-plan-farecalc__table-cell m-trip-plan-results__itinerary-fare-title m-trip-plan-farecalc__header">
-    Valid On
-  </div>
+<%= if @show_monthly_passes? do %>
```

Comment:
It's a view function, so wouldn't it be accessible from this template doing something like so:

```suggestion
<%= if SiteWeb.TripPlanView.show_monthly_passes?(@itinerary) do %>
```

Then you wouldn't need to compute `@show_monthly_passes?` to pass it in via that [render_to_string](https://github.com/mbta/dotcom/pull/571/commits/b45efcde94d20773f2157ed14d476d8b5e99564b#diff-5306341842cc891dd7f7f0c58435c22fR617) call.

Reply:
Great point! I updated both usages.